### PR TITLE
feat: Provide User with actionable error message when no tables are configured for syncing

### PR DESCRIPTION
**Pull request by bbernays**
#### Summary

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: scheduler/scheduler.go [hunk 1]
```diff
@@ -31,6 +31,8 @@ const (
 	otelName               = "schedule"
 )
 
+var ErrNoTables = errors.New("no tables specified for syncing, review `tables` and `skip_tables` in the your config file to specify at least one table to sync")
```
Comment (hermanschaaf) [inline scheduler/scheduler.go:34]:
```suggestion
var ErrNoTables = errors.New("no tables specified for syncing, review `tables` and `skip_tables` in your config and specify at least one table to sync")
```

File: schema/table.go [hunk 2]
```diff
@@ -520,6 +520,9 @@ func (t *Table) TableNames() []string {
 }
 
 func (t *Table) Copy(parent *Table) *Table {
+	if t == nil {
+		return nil
+	}
```
Comment (hermanschaaf) [inline schema/table.go:525]: Is this change for testing? Could we move the `if t == nil` logic to the test instead? I'm just wary that adding this here might hide future bugs

Comment (bbernays) [reply schema/table.go:525]: Definitely, we can move it, but currently it panics if you try to copy a nil table...

Comment (hermanschaaf) [reply schema/table.go:525]: > currently it panics if you try to copy a nil table...

Sounds reasonable to me, you shouldn't be trying to copy a nil table :)

**Pull request merged**